client/src/pages/about/index.js

```diff
@@ -48,7 +49,14 @@ const About = ({ data, preview = false }) => {
             }}
           />
         )}
-        <Hero content={hero && hero} />
+        {hero && (
+          <Hero
+            title={hero.title && hero.title}
+            subtitle={hero.subtitle && hero.subtitle}
+            mediaType={hero.mediaType && hero.mediaType}
+            link={hero.link && hero.link}
```

Comment:
```suggestion
            title={hero.title}
            subtitle={hero.subtitle}
            mediaType={hero.mediaType}
            link={hero.link}
```

Same here. If we want to defend ourselvs against undefined, we'll have to do that inside the `<Hero />`. Alternatively not renderingn the hero at all if those values does not exist